docs/src/frontend/index.md

```diff
@@ -696,6 +696,26 @@ acc.tokensAccepted() => Promise<Array<Token>>
 Returns a Promise for an array of tokens that are accepted by `acc`.
 On networks which do not keep track of this information (e.g. Ethereum), this returns an empty array.
 
+---
+@{ref("js", "appOptIn")}@{ref("js", "stdlib.accountAppOptIn")}
+```js
+acc.appOptIn(contract) => Promise<void>
+stdlib.accountAppOptIn(account, contract) => Promise<void>
```

Comment:
This one shouldn't be added because it is a write function

Reply:
So we only want to add `accountAppOptedIn` and not `accountAppOptIn`?

Reply:
I am saying that `stdlib.accountAppOptIn` should not exist because we generally don't have write functions in the `stdlib` namespace. Furthermore, I think that it shouldn't be in `acc` either. It should be in `ctc`, so... `acc.contract(info).optIn()`, because we need to do contract verification